### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by GbengaOshinaga**
Hey guys, so Expobook wasn't as awesome as I hoped it would be, so I implemented storybook using the suggestion given by @caiokf. Let me know what you think.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ module.exports = {
 		Promise: true,
 		window: true,
 		localStorage: true,
+		__DEV__: true,
```
Comment (caiokf) [inline .eslintrc.js:41]: maybe you dont need this anymore

**Comment by caiokf** [comment]
i like it. just make sure you can run yarn storybook and yarn start one after the other and the right app pops up. Sometimes expo caches the config, so if you run yarn storybook, then later yarn start, it would still show the storybook. If that works, i like it.

One more comment is that I usually prefer to not have base or partial components like `CenteredView`, or `EpisodeCover` on storybook, instead I focus more on Screens, like `PlayerScreen`, `ExploreScreen`, `PodcastListScreen`. Just because simple components tend to add a lot of extra effort to maintain, so I rather use storybook as a design tool, to check all screens without having to use the app to navigate.

**Comment by GbengaOshinaga** [comment]
I have refactored the stories to use screens instead of individual components. I'd advise that going forward, we should separate the screens into container and presentational components. A benefit of that applicable to this task is it makes it easier to create stories for screens in the future, as writing stories for connected components is problematic. For example, I couldn't write a story for the Player screen because its children are connected components, and they required data that could not be passed in. Separating them would simply allow us to be writing stories for the presentational components.

**Comment by caiokf** [comment]
@GbengaOshinaga agreed on separating presentational components. @nataphilips if you need i go through separating Player with you

**Pull request merged**